### DISCUSSION TITLE
ssl证书更新

### DISCUSSION THREAD

**Discussion by huanglian666** [1Panel-dev/1Panel#4965] [Q&A]
面板自带的网站目录下证书ssl自动更新证书以后，使用这个证书的网站会自动更新代理吗？还是说需要手动重新启动一下openrestry呢？ssl自动更新保存的证书位置和openresty的配置中使用的证书位置不一致

**Comment by zhengkunwang223** [comment]
会自动 reload

**Reply by huanglian666** [reply]
首先感谢您的回复
其次我还想问一下？这个reload需要我配置证书自动推送到网站下，SSL路径下面吗？如果是这样的话，我一个泛域名证书，如何推送多个地址呢？还是不需要我做任何设置，1panel面板就会自动给我更新好呢？
